.travis.yml

```diff
@@ -6,16 +6,10 @@ cache:
     directories:
         - $HOME/.composer/cache/files
 
-php:
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-
 env:
     global:
         - TEST_COMMAND="composer test"
+        - SYMFONY_PHPUNIT_VERSION="6.3"
```

Comment:
Did tests fail without exposing this environment variable? The bridge should pick the best PHPUnit version automatically if you don't do it.

Reply:
Yes, it did. It was some issues with mattiasnoback/config-test.

Reply:
Have you more details on those issues? I could look at it and maybe fix those in the library and so remove those variables from our travis file.